### ISSUE TITLE
Conduit version check is broken

### ISSUE THREAD

**Issue by mneilly-et** [LLNL/UMT#25]
The following version check is broken when the conduit minor version > 8. E.g. - 0.9.0 fails the check.

https://github.com/LLNL/UMT/blob/ed70b58e77b6dfb29b6b7f01d53bde2a02b7f218/src/teton/driver/test_driver.cc#L67-L70

Yet, the build script specifies 0.9.0 here:

https://github.com/LLNL/UMT/blob/b7867c4b05710c9d065b9fc5e16ca32972657948/build_and_run_umt.sh#L57

**Comment by BradWhitlock** [comment]
You are correct. Try the following in the meantime or always define _TETON_CONDUIT_HAS_TILED_FUNCTION_.

```
#define VERSION_NUMBER(MAJOR, MINOR, PATCH) (((MAJOR) * 100 * 100) + ((MINOR) * 100) + (PATCH))
#define CONDUIT_VERSION_NUMBER VERSION_NUMBER(CONDUIT_VERSION_MAJOR, CONDUIT_VERSION_MINOR, CONDUIT_VERSION_PATCH)

// Determine whether Conduit has the tiled function.
#if CONDUIT_VERSION_NUMBER >= VERSION_NUMBER(0, 8, 9)
#define TETON_CONDUIT_HAS_TILED_FUNCTION
#endif 
```

In newer versions of the code (not yet released) the Conduit version checks were removed since Conduit 0.9.0 was made the minimum version used for UMT.
